doc/userguide/rules/http-keywords.rst

```diff
@@ -590,6 +592,70 @@ Notes
 
 -  further notes at the ``file_data`` section below.
 
+http.server
```

Comment:
this is a copy of the file_data or http server body, and that makes it almost completely wrong. It's better to look at some of the other keywords registered using the same 'stub', for example http_accept and http_accept_enc.

Reply:
Updated documentation for `http.server` and `http.location`